sdks/python/apache_beam/io/external/kafka.py

```diff
@@ -64,11 +59,20 @@ class ReadFromKafka(ptransform.PTransform):
   byte_array_deserializer = 'org.apache.kafka.common.serialization.' \
                             'ByteArrayDeserializer'
 
+  _urn = 'beam:external:java:kafka:read:v1'
+
+  _schema = {
+    'consumer_config': typehints.Iterable[typehints.KV[unicode, unicode]],
```

Comment:
Let's use typing rather than typehints wherever we can. We should modify get_coder to accept this via native_type_conversion.convert_to_beam_type.

Reply:
Fixed.